Sources/KountDataCollector.xcframework/Info.plist

```diff
@@ -8,33 +8,32 @@
 			<key>HeadersPath</key>
 			<string>Headers</string>
 			<key>LibraryIdentifier</key>
-			<string>ios-arm64_x86_64-simulator</string>
```

Comment:
Did they change the name of the Library for the simulators?  Or are we taking out support for using simulators here?